first_hour_on_terra/working_with_data_collections.ipynb

```diff
@@ -0,0 +1,449 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Working with data collections\n",
+    "\n",
+    "<table align=\"left\">\n",
+    "\n",
+    "  <td>\n",
+    "    <a href=\"https://github.com/DataBiosphere/terra-axon-examples/blob/main/first_hour_on_terra/working_with_data_collections.ipynb\">\n",
+    "      <img src=\"https://cloud.google.com/ml-engine/images/github-logo-32px.png\" alt=\"GitHub logo\">\n",
+    "      View on GitHub\n",
+    "    </a>\n",
+    "  </td>\n",
+    "  <td>\n",
+    "    <a href=\"https://console.cloud.google.com/vertex-ai/workbench/deploy-notebook?download_url=https://github.com/DataBiosphere/terra-axon-examples/main/first_hour_on_terra/working_with_data_collections.ipynb\">\n",
+    "      <img src=\"https://lh3.googleusercontent.com/UiNooY4LUgW_oTvpsNhPpQzsstV5W8F7rYgxgGBD85cWJoLmrOzhVs_ksK_vgx40SHs7jCqkTkCk=e14-rj-sc0xffffff-h130-w32\" alt=\"Vertex AI logo\">\n",
+    "      Open in a Terra notebook instance\n",
+    "    </a>\n",
+    "  </td>                                                                                               \n",
+    "</table>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "tvgnzT1CKxrO"
+   },
+   "source": [
+    "## Overview\n",
+    "\n",
+    "This notebook provides examples of working with data collections in Terra. Build upon the best practices described in this notebook to create and share your own data collections. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "d975e698c9a4"
+   },
+   "source": [
+    "### Objective\n",
+    "\n",
+    "Perform common workspace resource operations including:\n",
+    "\n",
+    "1. Create a new data collection from cloud data.\n",
+    "1. Share the data collection with collaborators.\n",
+    "1. Add the data collection as a resource to a new workspace.\n",
+    "\n",
+    "#### How to run this notebook\n",
+    "\n",
+    "Please run the setup section before running any other section in this notebook.\n",
+    "\n",
+    "#### Costs\n",
+    "\n",
+    "This notebook takes less than a minute to run, which will typically cost less than $0.01 of compute time on your cloud environment."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "### Setup\n",
+    "\n",
+    "Run the cell below to capture the ID of the current workspace. You'll use this value to return to the current workspace after you've created a new workspace as part of the process of creating a data collection."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import json\n",
+    "from ipywidgets import Button, Layout\n",
+    "\n",
+    "CURRENT_WORKSPACE_ID = !terra workspace describe --format=json | jq --raw-output \".id\""
```

Comment:
I tend to prefer notebook cells to output **something**, just to show that it actually did something. Here you could add something like `print(f"Current workspace ID is {CURRENT_WORKSPACE_ID}")`

Reply:
fixed!